package.json

```diff
@@ -23,7 +23,7 @@
     "method-override": "2.3.3",
     "moment": "2.10.3",
     "passport": "0.2.2",
-    "passport-saml": "0.5.3",
```

Comment:
That'd be great if you could, and let me have some credentials which would work on the AFDS instance.

Thanks,
Philip